lib/compat/wordpress-6.1/class-wp-theme-json-6-1.php

```diff
@@ -1295,7 +1295,7 @@ protected function get_layout_styles( $block_metadata ) {
 			if ( $block_gap_value ) {
 				foreach ( $layout_definitions as $layout_definition_key => $layout_definition ) {
 					// Allow skipping default layout for themes that opt-in to block styles, but opt-out of blockGap.
-					if ( ! $has_block_gap_support && 'default' === $layout_definition_key ) {
+					if ( ! $has_block_gap_support && 'flex' !== $layout_definition_key ) {
```

Comment:
This makes it more explicit which fallback styles we'll be outputting.

To do: Update the other checks in this file and in the JS implementation to match.

Reply:
On second thought, I'll resolve this thread — by adding a `:where` rule to the block-level styles, this PR can support a future grid or other type of layout that should also be output when blockGap support isn't available.